addons/gamification/static/src/less/gamification.less

```diff
@@ -0,0 +1,50 @@
+img.o_kanban_badge_avatars {
+    width: 30px;
+    height: 30px;
+    padding-left: 0;
+    margin-top: 8px;
+    border-radius: 2px;
+    -box-shadow: 0 1px 2px rgba(0, 0, 0, 0.2);
+}
+.o_kanban_goal {
+    .o_goal_state_block {
+        width: 200px;
+        height: 130px;
+        margin: auto;
+        margin-bottom: -20px;
+        text-align: center;
+        .o_goal_state {
+            font-size: 2.5em;
+            font-weight: bold;
+            padding-top: 30px;
+            .fa {
```

Comment:
Instead of overriding standard fa class why not using fa-3x to make it bit bigger